### ISSUE TITLE
LV grid documentation

### ISSUE THREAD

**Issue by gplssm** [openego/ding0#130]
Write a documentation in plain English that provides the overview

**Comment by gplssm** [comment]
I did some restructuring and added some introductionary addings... 

I reworked existing text until [here](https://github.com/openego/dingo/blob/5da2ad80f381a303891421ee0a924cf185a4744e/doc/about.rst#L1-#L30).

Following TODOs remain

- [ ] Finish [section about residential LV grid branches](https://github.com/openego/dingo/blob/5da2ad80f381a303891421ee0a924cf185a4744e/doc/about.rst#L52-#L96) @EdithaK 
- [ ] Finish [section about retail/industrial and agricultural LV grid branches](https://github.com/openego/dingo/blob/5da2ad80f381a303891421ee0a924cf185a4744e/doc/about.rst#L98-#L101) @gplssm

**Comment by gplssm** [comment]
I added a description for the branches retail/industrial/agricultural. If you need, take inspiration how to use tables, equations, figures, and references to literature in sphinx docs.

**Comment by gplssm** [comment]
I'm done so far with a first version. @EdithaK you can close the issue when residential section is complete

**Comment by piaulous** [comment]
Deprecated due to new LV grid generation method, see #380.
Already mentioned #383.
